SensorApplication.java

```diff
@@ -0,0 +1,69 @@
+package sensor;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.border.TitledBorder;
+
+public class SensorApplication extends JFrame {
+	
+	public SensorApplication() {
+		setTitle("Sensor Tracker");
+		setLayout(new GridLayout(3,1));
+		
+        Sensor[] sensors = {
+        		new PressureSensorAdapter(new PressureSensor()),
+                new RadiationSensorAdapter(new RadiationSensor()),
+                new TemperatureSensorAdapter(new TemperatureSensor())
+        };
+        

```

Comment:
If Sensor instances get added to or removed from the Sensor array, this may be better to decrease the amount of hard-coding.
> setLayout(new GridLayout(sensor.length,1));

Reply:
I agree with @jerryzhu94, although you should take caution in this approach.  If new Sensor objects are added to sensors[] after GridLayout is created, it may not resize.  

I think an optimal approach would be: `setLayout(new GridLayout(0,1));` 
If you set the number of rows or columns in GridLayout to 0, it will not put a cap on that dimension's size, and  automatically expand in that direction.  In this example, it will be 1 column wide, adding more sensors below as needed.